src/main/java/it/eng/idsa/businesslogic/configuration/ApplicationConfiguration.java

```diff
@@ -27,6 +27,8 @@ public class ApplicationConfiguration {
 	private String openDataAppReceiver;
 	private String camelReceiverPort;
 	private String camelSenderPort;
+	private int wssSelfDescriptionPort;
```

Comment:
Do we need this property if we are going to calculate it's value based on some other field?